### PR TITLE
DOC: add manual notebook for lambda-K-Pi 

### PR DESCRIPTION
**Pull request by shenvitor**
Using the exact same model for Eta-Pi-P as a test for Lambda-K-Pi in manual and flexible symbolic `sympy` way.

### CODE REVIEW

**Comment by shenvitor** [comment]
Screenshot 2024-08-30 at 18 23 30
The Dalitz plot in this PR looks awesome rn 🍡  !
(hopefully it's correctly formulate, I will double-check later next week)

File: docs/lambda-k-pi/index.md [hunk 1]
```diff
@@ -3,4 +3,5 @@
 ```{toctree}
 ampform
 ampform-dpd
+manual-symbolic
```
Comment (redeboer) [inline docs/lambda-k-pi/index.md:6]: Why call this `manual-symbolic`, not `manual`?

**Review by redeboer — CHANGES_REQUESTED** [review]
I don't see the use of creating an (almost) exact copy of the eta pi p notebook. That one is more for playing around with a super simple amplitude model that has three subsystems and one resonance in each, where you can then see the effect of angular momentum. As we know, that model is not physical.

For $\Lambda K \pi$, we have to step up the game. This is supposed to be a model that formulates the model from the (preferably) `ampform-dpd` notebook manually, so that it can then be tweaked with a proper treatment of e.g. #58.

What is currently missing are multiple resonances, treatment of spin of the final-state particles, spin alignment, etc. That also makes the UI design not suitable. I would therefore start to try to reproduce the model (symbolic only!) from `ampform-dpd.ipynb`.

**Comment by shenvitor** [comment]
> I don't see the use of creating an (almost) exact copy of the eta pi p notebook. That one is more for playing around with a super simple amplitude model that has three subsystems and one resonance in each, where you can then see the effect of angular momentum. As we know, that model is not physical.
> 
> For $\Lambda K \pi$, we have to step up the game. This is supposed to be a model that formulates the model from the (preferably) `ampform-dpd` notebook manually, so that it can then be tweaked with a proper treatment of e.g. #58.
> 
> What is currently missing are multiple resonances, treatment of spin of the final-state particles, spin alignment, etc. That also makes the UI design not suitable. I would therefore start to try to reproduce the model (symbolic only!) from `ampform-dpd.ipynb`.

I think the idea here is to have multiple notebook for symbolic manually in different levels of complications, from simplest to some more complicated ones later  in this reaction section.

**Comment by redeboer** [comment]
> I think the idea here is to have multiple notebooks for symbolic manually in different levels of complications, from simplest to some more complicated ones later in this reaction section.

That's fine, but to have multiple copies of the same idea is just a maintenance burden. The 'manual' eta pi p notebook is just a pedagogical exercise: some **spinless** initial and final state ("p gamma to eta pi p" is even misleading) with **one** resonance per subsystem. In fact, the only difference between the notebooks now are the labels of those three imaginary resonances and the imaginary final state.

In the end, what we want to achieve is a **physical** description of $p\gamma \to \Lambda K \pi$, that is, with correct description of the production process, all dominating resonances in each subsystem, and a proper treatment of spin alignment of the final state. There is no generic package that can do that, so we need to go through a process of manually formulating each of these components to gain understanding of its behaviour.

**Comment by redeboer** [comment]
I'm fine with adding this notebook, but a lot needs to be done to this PR first, for instance:
- rotations for each decay node (see model from `ampform.ipynb` and `ampform-dpd.ipynb`
- a sum over an arbitrary number of resonances

**Comment by shenvitor** [comment]
Okay, I see your points. Yup we can make it more accurate than now.

**Comment by shenvitor** [comment]
Rejected by @redeboer, so close this as not planned for now, but we can still see the record if we need it in the future.

**Pull request closed without merging**